onboarding/src/main/res/values/strings.xml

```diff
@@ -17,7 +17,18 @@
     <string name="manual_server_clear_url">Clear server url</string>
     <string name="manual_server_wrong_url">Invalid Home\u00A0Assistant address</string>
     <string name="connection_screen_malformed_url">Verify that your Home\u00A0Assistant address is correct</string>
-    <string name="local_first_title">Local by default.\nRemote when you’re ready.</string>
+    <string name="local_first_title">Local by default.\nRemote when you\'re ready.</string>
     <string name="local_first_content">Your Home\u00A0Assistant is only accessible on your home network. To control your smart home from anywhere, you can set up remote access later in your settings.\n\nFor now, you\'re securely connected to your local network.</string>
     <string name="local_first_next">Got it</string>
+    <string name="location_sharing_title">Use this device\'s location for automations</string>
```

Comment:
This is one option, but you can also:
 - simply track the device
 - use different URLs
 - disable app lock when at home
It also doesn't immediately mean you get an automation based on device location.

We shouldn't cover all that with one title, but I think the current title is too limited.